curp/tests/server.rs

```diff
@@ -31,11 +29,14 @@ async fn basic_propose() {
             .unwrap(),
         vec![0]
     );
+
+    group.stop();
```

Comment:
can we stop the group when drop it

Reply:
Nope. Because we need to move the node to another thread and drop it.